packages/block-library/src/navigation/index.php

```diff
@@ -65,58 +65,58 @@ function block_core_navigation_sort_menu_items_by_parent_id( $menu_items ) {
 
 		return $menu_items_by_parent_id;
 	}
+}
 
-	/**
-	 * Turns menu item data into a nested array of parsed blocks
-	 *
-	 * @param array $menu_items               An array of menu items that represent
-	 *                                        an individual level of a menu.
-	 * @param array $menu_items_by_parent_id  An array keyed by the id of the
-	 *                                        parent menu where each element is an
-	 *                                        array of menu items that belong to
-	 *                                        that parent.
-	 * @return array An array of parsed block data.
-	 */
-	function block_core_navigation_parse_blocks_from_menu_items( $menu_items, $menu_items_by_parent_id ) {
-		if ( empty( $menu_items ) ) {
-			return array();
-		}
```

Comment:
I see there are still protected functions above. Are these still Gutenberg-only functions?

Reply:
No from what I see.

E.g. `block_core_navigation_sort_menu_items_by_parent_id` is called in `render_block_core_navigation` without garding `defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN`

Reply:
So potentially there are remaining code paths that can lead to errors right? Or are these impossible to trigger on Core @scruffian

Reply:
Yeah I think the others are guarded.